server/src/main/java/com/google/fhir/proxy/HttpFhirClient.java

```diff
@@ -63,6 +63,7 @@ public abstract class HttpFhirClient {
   // https://www.hl7.org/fhir/async.html
   // We should NOT copy Content-Length as this is automatically set by the RequestBuilder when
   // setting content Entity; otherwise we will get a ClientProtocolException.
+  // TODO(http://go/gh/google/fhir-access-proxy/issues/60): Allow Accept header
```

Comment:
nit: Please avoid go links as they are not publicly accessible.

Reply:
done